packages/extension-ui/src/Popup/Accounts/NewAccount.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ThemeProps } from '../../types';
+
+import React, { useCallback, useContext, useEffect, useMemo, useRef, useState } from 'react';
+import { RouteComponentProps, withRouter } from 'react-router';
+import styled from 'styled-components';
+
+import { AuthUrls } from '@polkadot/extension-base/background/handlers/State';
+import { AccountJson } from '@polkadot/extension-base/background/types';
+
+import { AccountContext, ActionContext, Button, PopupBorderContainer } from '../../components';
+import useToast from '../../hooks/useToast';
+import useTranslation from '../../hooks/useTranslation';
+import { getAuthList, updateAuthorization, updateAuthorizationDate } from '../../messaging';
+import { NewAccountSelection } from '../../partials';
+import { createGroupedAccountData } from '../../util/createGroupedAccountData';
+
+interface Props extends RouteComponentProps, ThemeProps {
+  className?: string;
+}
+
+const ButtonsGroup = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  padding-bottom: 0px;
+  position: absolute;
+  bottom: 16px;
+  left: 0px;
+  right: 0px;
+`;
+
+function NewAccount({ className, location: { search } }: Props): React.ReactElement<Props> {
+  const { hierarchy, selectedAccounts = [], setSelectedAccounts } = useContext(AccountContext);
+  const [authList, setAuthList] = useState<AuthUrls | null>(null);
+
+  const { t } = useTranslation();
+  const { show } = useToast();
+  const onAction = useContext(ActionContext);
+  const searchParams = new URLSearchParams(search);
+  const url = searchParams.get('url');
+  const { flattened } = useMemo(() => createGroupedAccountData(hierarchy), [hierarchy]);
+  const testRef = useRef<AccountJson[] | []>([]);
+
+  useEffect(() => {
+    getAuthList()
+      .then(({ list }) => setAuthList(list))
+      .catch((e) => console.error(e));
+  }, []);
+
+  useEffect(() => {
+    getAuthList()
+      .then(({ list }) => {
+        if (url && !list[url]) {
+          return;
+        }
+
+        if (url && setSelectedAccounts) {
+          setSelectedAccounts(list[url].authorizedAccounts);
+        }
+      })
+      .catch(console.error);
+  }, [setSelectedAccounts, url]);
```

Comment:
In both cases you are fetching authList. Seems like something is wrong with this flow 🤔